packages/bsp/common/etc/update-motd.d/10-armbian-header

```diff
@@ -14,7 +14,7 @@ MOTD_DISABLE=""
 
 [[ -f /etc/armbian-release ]] && . /etc/armbian-release
 [[ -f /etc/armbian-distribution-status ]] && . /etc/armbian-distribution-status
-[[ -f /etc/lsb-release && -f /etc/armbian-distribution-status ]] && DISTRIBUTION_CODENAME=$(cat /etc/lsb-release | grep CODENAME | cut -d"=" -f2) && DISTRIBUTION_STATUS=$(cat /etc/armbian-distribution-status | grep $DISTRIBUTION_CODENAME | cut -d"=" -f2)
```

Comment:
We are providing packages for older releases, where nothing has been changed. I would not remove this line. But add condition [[ -f /etc/os-release && ...

Reply:
OK thanks, got it. I re-added the original line and just added the new one..